spi/transfer-spi/src/main/java/org/eclipse/dataspaceconnector/spi/transfer/command/CommandHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.eclipse.dataspaceconnector.spi.transfer.command;
+
+import org.eclipse.dataspaceconnector.spi.ObjectFactory;
+
+/**
+ * CommandHandlers are ephemeral objects that receive a {@link Command} object and act on it.
+ * They must have an empty default constructor because they will get instantiated through reflection by the {@link ObjectFactory}. Also, they MUST
```

Comment:
I don't think it should be a requirement to have a default ctor or require that handler instances be instantiated reflectively via the Injection infrastructure. It should be possible to register handler instances in the runtime.

Reply:
it certainly is :)